bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -193,7 +193,8 @@ spec:
             --nodeport \
             --loglevel "${OVN_KUBE_LOG_LEVEL}" \
             ${hybrid_overlay_flags} \
-            --logfile /dev/stdout
```

Comment:
Can you please add this port (and the one below) to the pod spec, so we claim the port?